java/src/main/java/com/microsoft/graph/bulk/impl/model/GremlinVertexProperty.java

```diff
@@ -0,0 +1,23 @@
+package com.microsoft.graph.bulk.impl.model;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import java.util.UUID;
+
+@Data
+@EqualsAndHashCode
+public class GremlinVertexProperty {
+    //TODO: Evaluate impacts to changing this to be Serializable instead of just Object
```

Comment:
We should either remove these TODOs, or implement them if we have the bandwidth :)